js/src/views/Application/TabBar/tabBar.js

```diff
@@ -140,12 +190,115 @@ export default class TabBar extends Component {
         value={ isTest ? 'TEST' : netChain } />
       );
 
-    return this.renderLabel('status', bubble);
+    return this.renderLabel(label, bubble);
   }
 
   onActivate = (tab) => {
     const { router } = this.context;
+    const activeRoute = tab.props['data-route'];
```

Comment:
Why do we have to use `data` attributes here?
We could either:
1. Attach lambda for each `Tab::onActivate` (less performant?)
2. Wrap the Tab into another component (not sure if possible?)

I don't feel that `data-` attributes is the best way to do it.